scripts/set-environment

```diff
@@ -61,6 +61,7 @@ TERMINUS_ENV=${TERMINUS_ENV:-$DEFAULT_ENV}
 #=====================================================================================================================
 (
   echo 'export PATH=$PATH:$HOME/bin'
+  echo "export PATH='$HOME/.composer/vendor/bin:$PATH'"
```

Comment:
I'm not sure if this can run here or if it needs to run before `composer global require`

Reply:
DON'T USE COMPOSER GLOBAL REQUIRE!